pmd-core/src/main/java/net/sourceforge/pmd/RuleSetFactoryCompatibility.java

```diff
@@ -67,6 +67,11 @@ public RuleSetFactoryCompatibility() {
         // unused/UnusedModifier moved AND renamed, order is important!
         addFilterRuleMoved("java", "unusedcode", "unnecessary", "UnusedModifier");
         addFilterRuleRenamed("java", "unnecessary", "UnusedModifier", "UnnecessaryModifier");
+
+        // PMD 6.0.0
+        addFilterRuleMoved("java", "controversial", "unnecessary", "UnnecessaryParentheses");
+        addFilterRuleMoved("java", "unnecessary", "UnnecessaryParentheses", "UselessParentheses");
```

Comment:
I'm not sure we need this rename... I see no issue with the old name. It was actually consistent with the `java-unnecessary/UnnecessaryModifier` rule. Did you have any concerns with it?